### DISCUSSION TITLE
Type of Boundary Conditions

### DISCUSSION THREAD

**Discussion by APernatii** [PeriHub/PeriLab.jl#158] [Q&A]
Hallo,

I would like to point out, that I couldn't find the listing of available types of Boundary Conditions in documentation.

**Comment by JTHesse** [comment]
Hi @APernatii thank you for letting us know, can you please create a new [Documentation Request](https://github.com/PeriHub/PeriLab.jl/issues/new/choose) and we will add the information, thank you!
